bin/maya.bat

```diff
@@ -23,9 +23,11 @@ if "%MINDBENDER_ASSET%"=="" goto :missing_set_asset
 :: If user forgets to include task with "maya"..
 if "%1"=="" goto :missing_task
 
-set MINDBENDER_WORKDIR=%CD%\work\%1\%USERNAME%\maya
+if not exist "%1" goto :missing_taskfolder
+
+set MINDBENDER_WORKDIR=%CD%\%1\%USERNAME%\maya
 if Not exist %MINDBENDER_WORKDIR% (
-	echo Creating new task "%1"..
+	echo Creating folder for task "%1"..
```

Comment:
I changed it to not confuse the user that they used creation API (_new.bat_) and then when use that which they created it says creating your task again.